packages/EasyErrorHandler/src/Bridge/Laravel/Translator.php

```diff
@@ -24,9 +25,37 @@ public function __construct(IlluminateTranslatorInterface $decorated)
      */
     public function trans(string $message, array $parameters): string
     {
-        // TODO: rework after upgrading all the illuminate and laravel packages to ^6.0
-        $method = \method_exists($this->decorated, 'trans') ? 'trans' : 'get';
+        $translation = $this->doTrans($message, $parameters, false);
```

Comment:
Here is what I would do to keep things simple to understand/read.

- Delete the `processMessage()` method we don't need it
- Revert the changes made to `doTrans()` this method should have only 1 purpose, translate

And then that's how things would look like:

```php
$translation = $this->doTrans($message, $parameters);

if ($translation !== $message) {
	return $translation;
}

$namespacedMessage = \sprintf('%s::%s', BridgeConstantsInterface::TRANSLATION_NAMESPACE, $message);
$translation = $this->doTrans($namespacedMessage, $parameters);

// If translation is finally different we return it otherwise default to original message
return $translation !== $namespacedMessage ? $translation : $message; 
```